build/karma.conf.babel.js

```diff
@@ -44,9 +46,6 @@ module.exports = (karmaConfig) => {
           ...webpackConfig.module.loaders,
         ],
       },
-      plugins: [
-        new webpack.DefinePlugin(config.compiler_globals),
-      ],
```

Comment:
Cruft, we don't use globals in source code any longer.  Because we are a community package, we explicitly refer to `process.env` vars instead.